server/utils/azure-storage.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  BlobClient,
+  BlobServiceClient,
+  StorageSharedKeyCredential,
+} from "@azure/storage-blob";
+import { SessionUser } from "src/utils/types";
+
+const UPLOADS_CONTAINER_NAME = "uploads";
+
+export const blobService = new BlobServiceClient(
+  `https://${
+    process.env.AZURE_STORAGE_ACCOUNT as string
+  }.blob.core.windows.net`,
+  new StorageSharedKeyCredential(
+    process.env.AZURE_STORAGE_ACCOUNT as string,
+    process.env.AZURE_STORAGE_KEY as string
+  )
+);
+export const containerSerivce = blobService.getContainerClient(
```

Comment:
Spelling on this export.  I know, I am OCD.

Reply:
Good catch!